binding/HarfBuzzSharp.Shared/Blob.cs

```diff
@@ -1,34 +1,16 @@
 ﻿using System;
 using System.IO;
-using System.Runtime.InteropServices;
 
 namespace HarfBuzzSharp
 {
-	// public delegates
-	public delegate void BlobReleaseDelegate (object context);
```

Comment:
This will be a breaking change, so I think we should avoid this. It is a bit of a pain to have duplicate code, but it is what it is.

Reply:
Will revert it and introduce a duplicate